### DISCUSSION TITLE
do not block usage of chainlift when using powered launch

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#16619] [Feature requests]
When you use powered launch, chainlifts are disabled. I propose you allow chainlifts to be used on powered launch rides. I don't see a reason why this could be an issue

**Comment by TheAtomTwister** [comment]
I don't seem to have this problem, but I did notice that the construction UI is not coloring properly.  Here is an image of me building a lift hill while I'm using a powered launch operating mode on a B&M sitting coaster:
![Six Flags Great Adventure 2022-02-11 16-00-47](https://user-images.githubusercontent.com/97222164/153676483-7db17dc7-6a35-46d3-9a03-b21f899bbd47.png)

The construction UI should be translucent black, like this:
![Six Flags Great Adventure 2022-02-11 16-03-04](https://user-images.githubusercontent.com/97222164/153676592-33a8f191-a18c-4a26-972c-f403a873e303.png)

It does become the right color if I set the settings while the construction window is open, but if I close it and begin construction on something else, it opens in the default color.

If you're noticing that there is no lift hill on the LIM coaster, it is because the LIM coaster has no lift hill track pieces.
![Six Flags Great Adventure 2022-02-11 16-10-38](https://user-images.githubusercontent.com/97222164/153677347-68255b81-bd76-41a9-8eef-a912397c7465.png)

I am curious about your particular scenario and why you're not able to use lifts on launched coasters if they're not LIM coasters.

**Reply by 733737** [reply]
I actually might be thinking of reverse incline coasters, but also maybe since I always turn on build chainlift on all pieces, certain rides don't support chainlifts, so I am talking about coasters like LIM launched I suppose but I think it would be good also if they allowed boosters on all coasters even without sprites, so draw boosters on all coasters cheat, and use a generic booster sprite like from the looping coaster maybe with alternate added supports for rides that are more hefty I also would like launched lift hill on all coasters, and you could just use chainlift sprites as a stand in. I also might be thinking of rides that I modified to use powered launch that don't normally allow it, sometimes they might block chainlift for some reason



It would be nice also if you could have a reverse incline that then passes through the station and gets boosted as well so a powered launch reverse incline that supports a launched lift and regular chainlift, that ride would need lots of brakes.

**Reply by TheAtomTwister** [reply]
When the new track save format comes along, new track elements that support that kind of action could be added.  Alternatively, new operating modes that make stations act slightly differently might be a possibility.

**Reply by 733737** [reply]
maybe you could have a plugin that unlocks everything and adds in RMC Raptor track as a fallback, so you could put any piece on any coaster
